-------------------------------------------------------------------------------
Test set: DB_Test
-------------------------------------------------------------------------------
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.441 s <<< FAILURE! - in DB_Test
DB_Test  Time elapsed: 0.44 s  <<< ERROR!
java.lang.ExceptionInInitializerError
	at DB_Test.initHibirnate(DB_Test.java:15)
Caused by: org.hibernate.exception.JDBCConnectionException: Error calling Driver#connect
	at DB_Test.initHibirnate(DB_Test.java:15)
Caused by: org.postgresql.util.PSQLException: Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.
	at DB_Test.initHibirnate(DB_Test.java:15)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at DB_Test.initHibirnate(DB_Test.java:15)

